src/components/DataTable/DataTable.scss

```diff
@@ -103,20 +76,17 @@ $breakpoint: 768px;
   vertical-align: top;
 }
 
-.Cell-numeric {
-  text-align: right;
-}
-
-.Cell-fixed {
+.Cell-firstColumn {
   @include text-emphasis-normal;
   @include text-breakword;
-  position: absolute;
-  top: auto;
-  left: 0;
-  width: $fixed-column-width;
-  white-space: unset;
+  min-width: $first-column-width;
+  max-width: $first-column-width;
```

Comment:
Do you know why min and max work but not width?

Reply:
Good question! I think I originally took the set width out and had no min/max, but the first column very easily takes up the entire screen on small devices when the content is long, I was playing with adding either a max or a min but forgot to take it back out. What I'm thinking is that normal table behavior should be the default, and if implementations want to have set minimum or maximum we can always enhance with a prop.